lib/helpers.js

```diff
@@ -131,14 +131,19 @@ exports.generateRootCapability = async ({
   };
 };
 
-exports.getInvocationTarget = ({host, url}) => {
+exports.getInvocationTarget = ({url}) => {
   // look for `/kms/keystores/<keystoreId>/zcaps/`
   const cfg = config['kms-http'];
-  const baseKeystoreUrl = `https://${host}${cfg.routes.basePath}/keystores/`;
-  let idx = url.indexOf(baseKeystoreUrl);
-  if(idx !== 0) {
+
+  // FIXME: is the an acceptable way to account for the unknown namespace
+  // prefix? Cannot pass namespaceId in directly here because the request
+  // passes through documentLoader and webkms-switch
+  const matchToken = `${cfg.routes.basePath}/keystores/`;
+  let idx = url.indexOf(matchToken);
+  if(idx === -1) {
     return null;
   }
+  const baseKeystoreUrl = url.substr(0, idx + matchToken.length);
```

Comment:
I suspect all of this can be simplified somehow ... and it doesn't even map to how we want to use ezcap-express anymore, i.e., we need a breaking change with the removal of the `/zcaps` stuff and replacement with `urn:zcap:root:` (or whatever the prefix is) instead.

I would think we still want to compare against the host from the config. Really all this code here should just be about getting the keystore ID, right? It seems that the whole function should be rewritten with that in mind, unless I'm missing something. That would be easier to reason about too -- we just have some good code that ensures we can parse a valid keystore ID out of the URL -- and then tack on the path that follows that.